CHANGES.rst

```diff
@@ -56,7 +56,7 @@ There's a frood who really knows where his towel is.
 - Remove old upgrade steps.
   [wesleybl]
 
-- Add suport to Python 3.6, 3.7 and 3.8.
+- Add suport to Python 3.7 and 3.8.
```

Comment:
Please don't change the history. :-)

Reply:
I changed it because the Python 3.6 add support was in the changes of an ungenerated release. So from a new release point of view, it will be without Python 3.6 support.

Anyway, I undo the change and add a new entry.

Reply:
Ah, sorry, I missed that.